### DISCUSSION TITLE
Anyone knows what stack (zeit/vercel)'s built on ?

### DISCUSSION THREAD

**Discussion by barelyhuman** [vercel/vercel#4521] [General] [answered]
I have a feeling the dashboard is built on next.js but I don't really know what their api's are built on. 
Could be on next.js as well but would like to know, just out of curiosity.

sorry if a similar post already exists, I wasn't able to find one.

**Comment by Timer** [comment, marked as answer]
Nearly all of Vercel is powered by Next.js, including its APIs via [Next.js API Routes](https://nextjs.org/docs/api-routes/introduction).

**Reply by barelyhuman** [reply]
Oh nice, explains the dashboard rendering at that speeds. Thanks, @Timer 
btw, probably said too often on here but thanks for the great products 👍
